projects/plugins/jetpack/modules/masterbar/admin-color-schemes/colors/coffee/colors.scss

```diff
@@ -17,5 +17,13 @@ $color-primary-light: $studio-orange-30; // Calypso --color-primary-light
 $color-surface: $studio-white; // Calypso --color-surface
 $color-neutral-100-rgb: $studio-gray-100-rgb; // Calypso --color-neutral-100-rgb
 
+// Sidebar Nudges
+$menu-nudge-background: $studio-white;
+$menu-nudge-text-color: $studio-black;
+$menu-nudge-cta-background: $highlight-color;
+$menu-nudge-cta-color: $studio-white;
+$menu-nudge-cta-background-hover: rgb(138, 77, 0);
```

Comment:
Is this not the same as `$studio-orange-60`?

Reply:
Indeed! e8b68bc